config/routes.rb

```diff
@@ -22,6 +22,8 @@
     get "/delete", action: :confirm_destroy
   end
 
+  get "completed", to: "projects#completed"
```

Comment:
I'm not 100% convinced this is the best route. Would `projects/completed` make more sense?

Reply:
I think it would make more sense!